### PR TITLE
rename and fix text

### PR DESCRIPTION
**Pull request by eyalkraft**
## What does this PR do?

Rename our integration package and fix the looks.

## Related issues

- Closes https://github.com/elastic/security-team/issues/2803

## Screenshots

Search
Screen Shot 2022-01-30 at 19 01 59

Preview
Screen Shot 2022-01-30 at 18 29 51

Installation
Screen Shot 2022-01-30 at 18 49 23

Options expanded
Screen Shot 2022-01-30 at 18 50 08

### CODE REVIEW

**Comment by tehilashn** [comment]
@eyalkraft :
1. I think we should call it "CIS Kubernetes Vanilla Benchmark"
2. Not sure what's the reason to having the "Enable CIS Kubernetes Benchmark" option, what will happen if we turn it off?

**Comment by eyalkraft** [comment]
1. I followed the [Figma](https://www.figma.com/file/77qnvIhSgwNeMdLj7iylCE/CSP-Integration-Management?node-id=178%3A213674) provided by Einat. Also since "Vanilla Kuberenetes" isn't an official term or a real Kubernetes version I think we should avoid using it. Anyways this could be changed later if we decide.
2. What happens is our beat will not be active. There is no possibility to get rid of this configuration option (this exists for every integration). This is also present in the Figma.

**Review by tehilashn — APPROVED** [review]

File: packages/cis_kubernetes_benchmark/data_stream/findings/fields/base-fields.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  description: Data stream type.
+- name: data_stream.dataset
+  type: constant_keyword
+  description: Data stream dataset.
+- name: data_stream.namespace
+  type: constant_keyword
+  description: Data stream namespace.
+- name: '@timestamp'
+  type: date
+  description: Event timestamp.
```
Comment (CohenIdo) [inline packages/cis_kubernetes_benchmark/data_stream/findings/fields/base-fields.yml:12]: checkout this [PR](https://github.com/build-security/integrations/pull/3/files), 
doing a similar job. let's see how we don't repeat the same work.

Comment (eyalkraft) [reply packages/cis_kubernetes_benchmark/data_stream/findings/fields/base-fields.yml:12]: Hey @CohenIdo, this is just autogenerated by the `elastic-package create data-stream`.
Feel free to change it on your PR, I still think it's good to merge it that way so we don't miss some important autogenerated fields.

**Review by CohenIdo — APPROVED** [review]

**Pull request merged**